.github/workflows/dart_ci.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [ 2.7.2, 2.13.4, stable, dev ]
+        sdk: [ 2.7.2, 2.13.4 ]
```

Comment:
Looking for sanity check - I am making an assumption that the backpatch release line will not be usable on stable/dev.

Reply:
Yeah, this is reasonable 👍